packages/amplify-graphql-transformer-core/src/cdk-compat/stack-synthesizer.ts

```diff
@@ -37,6 +32,22 @@ export class TransformerStackSythesizer extends LegacyStackSynthesizer {
     );
   }
 
+  protected synthesizeTemplate(session: ISynthesisSession, _?: string): FileAssetSource {
+    const stack = this.boundStack;
```

Comment:
this method replaced `synthesizeStackTemplate` in runtime.

Reply:
What does replace mean? Is this a copy/clone of some CDK method? 
Also, I don't see any usages of this new method. Is this a place holder?